webapp/views/wpt-results.js

```diff
@@ -497,13 +530,9 @@ class WPTResults extends WPTColors(WPTFlags(PathInfo(LoadingState(TestRunsUIBase
 
   constructor() {
     super();
-    this.openAmendMetadata = (i, node) => {
-      return () => {
-        const amend = this.$.amend;
-        amend.test = node.path;
-        amend.productIndex = i;
-        amend.open();
-      };
+    this.openAmendMetadata = () => {
+      const amend = this.$.amend;
+      amend.open();
```

Comment:
```suggestion
      this.$.amend.open();
```

In fact, now that `openAmendMetadata` does not take any parameters, you can define this as a class method and use `on-click` instead.